src/CTA.Rules.Update/ActionsRewriter.cs

```diff
@@ -195,27 +195,35 @@ public override SyntaxNode VisitInvocationExpression(InvocationExpressionSyntax
             var symbols = _semanticModel.GetSymbolInfo(node);
             var newNode = (InvocationExpressionSyntax)base.VisitInvocationExpression(node);
 
-            foreach (var action in _fileActions.InvocationExpressionActions)
+            var symbol = symbols.Symbol;
+
+            if (symbol != null)
             {
```

Comment:
Optional: replace the `if` block with a guard statement to reduce nesting

Example: 
```csharp
if (someCondition)
{
    return newNode;
}

// do stuff down here

```

Reply:
Thanks, will follow this rule from now on.